.github/workflows/main.yml

```diff
@@ -132,6 +132,7 @@ jobs:
           - "wehi"
           - "wustl_htcf"
           - "xanadu"
+          - "ChristopherBarrington"
```

Comment:
Are you a profile @ChristopherBarrington ?

Reply:
```suggestion
```

Reply:
🤣 I shouldn't be! It should only have been there for the testing part though, right?

Reply:
I don't think you even need that for testing

Reply:
> In order to ensure that the config file is tested automatically with GitHub Actions please add your profile name to the profile: scope

Of course that isn't MY profile name 🤦 I thought it was weird